blog.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>OUR BLOG</title>
+</head>
+
+<body>
+  <header>
+    <a href="index.html">
+      <img src="..." alt="Binabox Logo" />
+    </a>
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="blog.html">Blog</a></li>
+        <li><a href="...">Catalog</a></li>
+        <li><a href="...">Collection</a></li>
+      </ul>
+      <input type="search" placeholder="Search..." />
+      <a href="...">Discord</a>
+      <a href="...">Connect</a>
+    </nav>
+  </header>
+
+  <main>
+    <section>
+      <!--Banner Title-->
+      <h2>Our blog</h2>
+      <div>
+        <ul>
+          <!--Group Buttons-->
+          <li><a href="index.html" target="_blank">Home</a></li>
+          <li><a href="blog.html" target="_blank">Our blog</a></li>
+          <li><a href="#" target="_blank">The Basilisks NFT</a></li>
+        </ul>
+      </div>
+    </section>
+
+    <aside>
+      <!-- Categories Section -->
+      <nav>
+        <h3>Categories</h3>
+        <ul>
+          <li><a href="#">Our blog (12)</a></li>
+          <li><a href="#">Metaverse (24)</a></li>
+          <li><a href="#">Gaming (24)</a></li>
+          <li><a href="#">Art (24)</a></li>
+          <li><a href="#">Robot (24)</a></li>
+          <li><a href="#">Launchpad (24)</a></li>
+          <li><a href="#">Crypto (410)</a></li>
+        </ul>
+      </nav>
+
+      <!-- Tags Section -->
+      <section>
+        <h3>Tags</h3>
+        <ul>
+          <li><a href="#">NFT</a></li>
+          <li><a href="#">ICO</a></li>
+          <li><a href="#">Robot</a></li>
+          <li><a href="#">Crypto</a></li>
+          <li><a href="#">Launchpad</a></li>
+          <li><a href="#">Metaverse</a></li>
+        </ul>
+      </section>
+    </aside>
+    <article>
+      <img src="basilisk-image.jpg" alt="The Basilisk" />
+      <p>Photo caption.</p>

```

Comment:
Use `<figure>` and `<figcaption>` for the image and its caption, which is the correct semantic approach.
read more about figure [<figure>: The Figure with Optional Caption element - HTML: HyperText Markup Language | MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure)

Reply:
It's not fixed :(